op-test-sequencer/sequencer/backend/work/builders/standardbuilder/config.go

```diff
@@ -0,0 +1,109 @@
+package standardbuilder
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/endpoint"
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/backend/work"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/seqtypes"
+)
+
+type Config struct {
+	// L1 execution-layer RPC endpoint
+	L1EL endpoint.MustRPC `yaml:"l1EL,omitempty"`
+
+	// L2 execution-layer RPC endpoint
+	L2EL endpoint.MustRPC `yaml:"l2EL,omitempty"`
+	// L2 consensus-layer RPC endpoint
+	L2CL endpoint.MustRPC `yaml:"l2CL,omitempty"`
```

Comment:
`omitempty` seems to contradict `MustRPC`. I think we could remove the `omitempty`s.